src/lib/errors.ts

```diff
@@ -1,66 +1,22 @@
-import {routeList} from '../routeList'
-
-export const enum ErrorCode {
-  Unknown = 'Unknown',
-
-  // actions/account/index
-  WalletInitFailed = 'WalletInit',
-  SaveClaimFailed = 'SaveClaim',
-  SaveExternalCredentialFailed = 'SaveExtCred',
-
-  // actions/sso/authenticationRequest
-  AuthenticationRequestFailed = 'AuthRequest',
-  AuthenticationResponseFailed = 'AuthResponse',
-
-  // actions/sso/paymentRequest
-  PaymentRequestFailed = 'PayRequest',
-  PaymentResponseFailed = 'PayResponse',
-
-  // actions/sso/index
-  CredentialOfferFailed = 'CredOffer',
-  CredentialsReceiveFailed = 'CredsReceive',
-  CredentialRequestFailed = 'CredRequest',
-  CredentialResponseFailed = 'CredResponse',
-  ParseJWTFailed = 'ParseJWT',
-
-  // actions/registration
-  RegistrationFailed = 'Registration',
-}
-
-// NOTE: these strings are localized, remember to update locale files
-const errorMessages: { [key in ErrorCode]: string } = {
-  [ErrorCode.Unknown]: 'Unknown Error',
-
-  [ErrorCode.WalletInitFailed]: 'Unable to initialize wallet',
-  [ErrorCode.SaveClaimFailed]: 'Could not save claim',
-  [ErrorCode.SaveExternalCredentialFailed]:
-    'Could not save external credential',
-
-  [ErrorCode.AuthenticationRequestFailed]: 'Authentication request failed',
-  [ErrorCode.AuthenticationResponseFailed]: 'Authentication response failed',
-  [ErrorCode.PaymentRequestFailed]: 'Payment request failed',
-  [ErrorCode.PaymentResponseFailed]: 'Payment response failed',
-
-  [ErrorCode.CredentialOfferFailed]: 'Credential offer failed',
-  [ErrorCode.CredentialsReceiveFailed]: 'Could not receive credentials',
-  [ErrorCode.CredentialRequestFailed]: 'Credential request failed',
-  [ErrorCode.CredentialResponseFailed]: 'Credential response failed',
-  [ErrorCode.ParseJWTFailed]: 'Could not parse JSONWebToken',
-
-  [ErrorCode.RegistrationFailed]: 'Registration failed',
-}
+import { routeList } from '../routeList'
+import strings from '../locales/strings'
+import ErrorCode from './errorCodes'
```

Comment:
can you re-export `ErrorCode` from this file to simplify all the imports back to just being
`import { AppError, ErrorCode } from 'src/lib/errors'`